www/convertUtils.js

```diff
@@ -0,0 +1,61 @@
+/*
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+*/
+
+var utils = require('cordova/utils');
+
+module.exports = {
+    /**
+    * Converts primitives into Complex Object
+    * Currently only used for Date fields
+    */
+    toCordova: function (contact) {
```

Comment:
Are we anticipating adding more conversions to these functions like the comments indicate?

Also, small nitpick: it would be nice to rename these to something a little more descriptive, like `toCordovaFormat` and `toNativeFormat`. `toCordova` reads a little strangely to me.